eth_tester/backends/pyevm/main.py

```diff
@@ -119,14 +120,14 @@ def get_default_account_keys():
 
 
 @to_dict
-def generate_genesis_state(account_keys):
+def generate_genesis_state(account_keys, overrides=None):  # TODO: Merge genesis state overrides
```

Comment:
If you don't intend to do the genesis state changes in this PR, can you:

- remove the `TODO`
- open an issue (suggesting this approach)
- remove this `overrides` keyword arg, which implies that the feature works